machine-learning/tutorials/SentimentAnalysis/Program.cs

```diff
@@ -134,117 +128,120 @@ public static void Evaluate(MLContext mlContext, ITransformer model)
+        private static void UseModelWithSingleItem(MLContext mlContext, ITransformer model)
         {
-            // <Snippet17>
-            var predictionFunction = model.CreatePredictionEngine<SentimentData, SentimentPrediction>(mlContext);
-            // </Snippet17>
+            // <SnippetCreatePredictionEngine1>
+            PredictionEngine<SentimentData, SentimentPrediction> predictionFunction = model.CreatePredictionEngine<SentimentData, SentimentPrediction>(mlContext);
+            // </SnippetCreatePredictionEngine1>
 
-            // <Snippet18>
+            // <SnippetCreateTestIssue1>
             SentimentData sampleStatement = new SentimentData
             {
-                SentimentText = "This is a very rude movie"
+                SentimentText = "This was a very bad steak"
             };
-            // </Snippet18>
+            // </SnippetCreateTestIssue1>
 
-            // <Snippet19>
+            // <SnippetPredict>
             var resultprediction = predictionFunction.Predict(sampleStatement);
-            // </Snippet19>
-            // <Snippet20>
+            // </SnippetPredict>
+            // <SnippetOutputPrediction>
             Console.WriteLine();
             Console.WriteLine("=============== Prediction Test of model with a single sample and test dataset ===============");
 
             Console.WriteLine();
-            Console.WriteLine($"Sentiment: {sampleStatement.SentimentText} | Prediction: {(Convert.ToBoolean(resultprediction.Prediction) ? "Toxic" : "Not Toxic")} | Probability: {resultprediction.Probability} ");
+            Console.WriteLine($"Sentiment: {sampleStatement.SentimentText} | Prediction: {(Convert.ToBoolean(resultprediction.Prediction) ? "Positive" : "Negative")} | Probability: {resultprediction.Probability} ");
+
             Console.WriteLine("=============== End of Predictions ===============");
             Console.WriteLine();
-            // </Snippet20>
+            // </SnippetOutputPrediction>
         }
 
-        public static void PredictWithModelLoadedFromFile(MLContext mlContext)
+        public static void UseLoadedModelWithBatchItems(MLContext mlContext)
         {
             // Adds some comments to test the trained model's predictions.
-            // <Snippet26>
+            // <SnippetCreateTestIssues>
             IEnumerable<SentimentData> sentiments = new[]
             {
                 new SentimentData
                 {
-                    SentimentText = "This is a very rude movie"
+                    SentimentText = "This was a horrible meal"
                 },
                 new SentimentData
                 {
-                    SentimentText = "I love this article."
+                    SentimentText = "I love this spaghetti."
                 }
             };
-            // </Snippet26>
+            // </SnippetCreateTestIssues>
 
-            // <Snippet27>
+            // <SnippetLoadModel>
```

Comment:
For consistency with SaveModelAsFile, consider moving this logic into its own method.

Reply:
Ok, that will be a good rev. I'll add an issue to track it. Would rather go ahead and get it out.

Reply:
Sounds good.